sapbtp-operator-charts/templates/deployment.yml

```diff
@@ -20,6 +22,9 @@ spec:
         {{- $secretTls := (include (print $.Template.BasePath "/secret-tls.yml") .) -}}
         {{- $configSha := (print $configmap $secret $secretTls) | sha256sum }}
         checksum/config: {{ $configSha }}
+        {{- if .Values.manager.annotations }}
+        {{- toYaml .Values.manager.annotations | nindent 8 }}
```

Comment:
how is this used? should a new entry be added to the values.yaml?

Reply:
this is just allowing consumers of this chart to provide custom annotations. In kyma, we use this to disable isto injection.